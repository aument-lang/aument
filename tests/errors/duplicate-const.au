const a = 10;
const a = 20;
