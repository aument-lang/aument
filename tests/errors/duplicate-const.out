parser error(14) in -: this constant is already declared
2 | const a = 20;
          ^
